templates/editProblem.tpl

```diff
@@ -46,8 +46,8 @@
           </a>
         </li>
         <li role="presentation">
-          <a href="#tagsTab" aria-controls="tagsTab" role="tab" data-toggle="tab">
-            {"tags"|_}
+          <a href="#metadataTab" aria-controls="metadataTab" role="tab" data-toggle="tab">
+            {"metadata"|_}
```

Comment:
We'll think of something more appropriate. Real people (the author) are not metadata. :-)